.golangci.yml

```diff
@@ -10,15 +10,22 @@ linters:
     - performance
     - sql
     - module
-    - unused
+    - format
+    - import
```

Comment:
These two (format, import) actually enables "go fmt" and "goimports" chekcing

The unused was accidentally added, I thought it is some kind of "unused code" check while it is actually a deprecated group.